src/instrumenter/instrumentation_context.ts

```diff
@@ -155,6 +186,14 @@ export class InstrumentationContext {
         this.scratchField = this.nameGenerator.getFresh("__mstore_scratch__", true);
         this.checkInvsFlag = this.nameGenerator.getFresh("__scribble_check_invs_at_end", true);
         this.utilsContractName = this.nameGenerator.getFresh("__scribble_ReentrancyUtils", true);
+        this.varInterposingQueue = dedup(
+            _varInterposingQueue,
+            (x: [VariableDeclaration, AbsDatastructurePath]) =>
+                `${x[0].name}_${x[1].map((x) => (x === null ? "[]" : x)).join("_")}`
```

Comment:
`x` is overlaped in arrow function `(x) => (x === null ? "[]" : x)` - this adds cognitive load. Is it possible to change variable name to something else?

Reply:
Fixed in a97c7d9.